models/public/retinaface-resnet50/model.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  RetinaFace-R50 is a medium size model with ResNet50 backbone for Face Localisation. It can output face bounding boxes and five facial landmarks in a single forward pass. More details provided in the paper <https://arxiv.org/abs/1905.00641>
```

Comment:
Please, use this [script](https://github.com/eizamaliev/OMZ_desrtiption_updater) to update description

Reply:
done